pennylane/math/quantum.py

```diff
@@ -455,6 +455,71 @@ def reduced_dm(state, indices, check_state=False, c_dtype="complex128"):
     return density_matrix
 
 
+def purity(state, indices, check_state=False, c_dtype="complex128"):
+    r"""Computes the purity from a state vector or density matrix.
+
+    .. math::
+        \gamma = Tr(\rho^2)
+
+    where :math:`\rho` is the density matrix. The purity of a normalized quantum state satisfies
+    :math:`\frac{1}{d} \leq \gamma \leq 1`, where :math:`d` is the dimension of the Hilbert space.
+    A pure state has a :math:`\gamma` of 1.
+
+    It is possible to compute the purity of a sub-system from a given state. To find the purity of
+    the overall state, include all wires in the "indices" argument.

```

Comment:
By conventional, we use `code formatting` to specify names of arguments.
```suggestion
    the overall state, include all wires in the ``indices`` argument.
```

Reply:
If I remember correctly, in latex, double quotes are coded like ``...'', is this the case for docstring here?